application/src/pages/Cart/CartForm/CartFormHeader/CartFormHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { NavLink } from 'react-router-dom';
+import { PATH } from '../../../PageRoutes/PageRoutes';
+import { CartFormPropTypes } from '../CartForm';
+import style from './CartFormHeader.module.css';
+
+export const CartFormHeader: React.FC<CartFormPropTypes> = ({
+  orderNum,
+  total
+}) => {
+  const { t } = useTranslation();
+  return (
+    <div className={style.title}>
+      <p>{`${t('cart.order.totalAmount')} ${total} $`}</p>
+      <div>
+        {orderNum.length > 1 &&
```

Comment:
Please create variable for `orderNum.length > 1`

Reply:
What is 1?

Reply:
it was my mistake, i fixed and refactor